### ISSUE TITLE
looking up a class from a map pulls the map into the class attribute

### ISSUE THREAD

**Issue by OliverM** [r0man/sablono#130]
I originally filed this issue on [Rum](https://github.com/tonsky/rum/issues/97) but @martinklepsch suggested it should be filed on sablono as an issue with `sablono.util/join-classes`. 

If I define a map of styles like so:

```clj
(def csslib
      {:table-header-row "bg-mid-gray"
       :table-header-cell "bg-yellow"
       :table-row "bg-red"
       :table-cell "bg-blue"
    })
```

and then try to refer to them as class attributes in a Rum `defc` call (assuming Rum's in your namespace under `r`) like so:

```clj
(r/defc table-cell < r/static [{:keys [text] :as cell}]
   [:td {:class (:table-cell csslib)}
    [:span text]])
```

The assigned class for the table cell in the generated html reads

```html
class="table-cell :table-header-row bg-mid-gray :table-header-cell bg-yellow :table-row bg-red :table-cell bg-blue"
```

One workaround is to put the map lookup inside a `str` call, which gives the expected class attribute of `class="bg-blue"` in the rendered html, and [Martin pointed out that](https://github.com/tonsky/rum/issues/97#issuecomment-245884226) other typical map lookups work fine:
```clj
(get csslib :table-cell)
(-> csslib :table-cell)
(csslib :table-cell)
```

**Comment by martinklepsch** [comment]
Here's a minimal example using just Sablono's `html` macro:
```
(html [:div {:class (:table-cell csslib)}
  [:span "abc"]]))
```

**Comment by r0man** [comment]
I am AFK at the moment. Patch with test welcome.

**Comment by r0man** [comment]
@OliverM and @martinklepsch I added a fix for this issue here https://github.com/r0man/sablono/pull/131
Could you try `0.7.5-SNAPSHOT` and let me know if this fixes the problem?

**Comment by r0man** [comment]
This should be fixed now. Released as `0.7.5`.
